website/blog/2023-08-14-mongodb-sorting-scalar.md

```diff
@@ -0,0 +1,166 @@
+---
+slug: mongodb-sorting-scalar
+title: How MongoDB Sorting Works for Scalar Values
+authors: [chi]
+description: >
+  In this blog post, we explore how MongoDB sorting works for scalar values.
+keywords:
+  - MongoDB sorting
+  - BSON comparison order
+image: /img/blog/post-cover-image.jpg
+tags: [community, product, tutorial]
+---
+
+![MongoDB sorting works for scalar values](/img/blog/mongodb-sorting-scalar.jpg)
+
+In this blog post, we explore how MongoDB sorting works for scalar values.
+
+<!--truncate-->
+
+Sorting compares BSON values to determine which value is equal, greater or less than the other to order them in the ascending or descending order.
```

Comment:
```suggestion
Sorting compares BSON values to determine which value is equal, greater or less than the other to return them in the ascending or descending order.
```

Maybe "to return them" or "to arrange them", so we don't have the word "order" twice?